unit_tests/test_seed_selector.py

```diff
@@ -114,7 +114,7 @@ def test_reflector_node_not_exists(self):
         self.setup_cluster(nodes_number=1)
         self.cluster.set_test_params(seeds_selector='reflector', seeds_num=1, db_type='scylla')
         sdcm.cluster.SCYLLA_YAML_PATH = os.path.join(os.path.dirname(__file__), 'test_data', 'scylla.yaml')
-        self.assertRaisesRegex(expected_exception=RetryError,
-                               expected_regex='Waiting for seed is selected by reflector',
-                               callable_obj=self.cluster.set_seeds,
+        self.assertRaisesRegex(RetryError,
```

Comment:
why did you remove argument names?

Reply:
`callable_obj` parameter was removed in 3.8
and can't do unnamed argument after named ones, so needed to take out the other too